---
geometry_msgs/Quaternion init_orientation
float64[3] gyro_bias
